patches/chrome-browser-ui-webui-settings-people_handler.cc.patch

```diff
@@ -0,0 +1,45 @@
+diff --git a/chrome/browser/ui/webui/settings/people_handler.cc b/chrome/browser/ui/webui/settings/people_handler.cc
+index 57cd1e34ebbc19681692a7e4100677ab0510daea..bab528521f520df2a6feb2861101fab39bc7eef6 100644
+--- a/chrome/browser/ui/webui/settings/people_handler.cc
++++ b/chrome/browser/ui/webui/settings/people_handler.cc
+@@ -89,6 +89,7 @@ struct SyncConfigInfo {
+ };
+ 
+ bool IsSyncSubpage(const GURL& current_url) {
++  BRAVE_IS_SYNC_SUBPAGE
+   return current_url == chrome::GetSettingsUrl(chrome::kSyncSetupSubPage);
```

Comment:
`GetSettingsUrl` looks like something we could easily override

Reply:
fixed in https://github.com/brave/brave-core/pull/5294/commits/b2fdf8c10dd8d42e0f3add64c775cfd12b8abd57